queue_fetcher/tasks/base.py

```diff
@@ -37,6 +37,7 @@ class QueueFetcher(object):
     """Deals with fetching things from an Amazon SQS queue
     """
     queue = None
+    region = 'eu-west-1'
```

Comment:
Can you add a `get_region()` method here? It's useful to do the same with `queue` above - maybe `get_queue_name()`?